src/styles/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  gap: 1px;
+  justify-content: left;
+  width: 706px;
+  padding: 10px;
+  flex-wrap: wrap;
+
+  & :nth-child(n) {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    min-width: 100px;
+    min-height: 100px;
+    background-color: #eee;
+    border: 1px solid black;
+    box-sizing: border-box;
+
+    &:hover {
+      cursor: pointer;
+      background-color: pink;
+      transition: transform 0.5s;
```

Comment:
Animations for both entering and leaving hover should be smooth, currently it's animated only on enter. Can you think of something that could be changed here to allow this to happen?

Reply:
Hi @EdPirro, Yes certanly. I'll put the transition property in the class selector above this one.